### ISSUE TITLE
[Crash] Error: ITO_Loading suspended while rendering, but no fallback UI was specified.Add a ...

### ISSUE THREAD

**Issue by yisiliu** [DimensionDev/Maskbook#2329]
I was ________, then Mask report an error.

> ITO_Loading suspended while rendering, but no fallback UI was specified.

Add a <Suspense fallback=...> component higher in the tree to provide a loading indicator or placeholder to display.

Error stack:
<pre>Error: ITO_Loading suspended while rendering, but no fallback UI was specified.

Add a <Suspense fallback=...> component higher in the tree to provide a loading indicator or placeholder to display.
    at Rj (chrome-extension://jkoeaghipilijlahjplgbfiocjhldnap/js/npm.react-dom.js:326:288)
    at Sj (chrome-extension://jkoeaghipilijlahjplgbfiocjhldnap/js/npm.react-dom.js:328:319)
    at Mj (chrome-extension://jkoeaghipilijlahjplgbfiocjhldnap/js/npm.react-dom.js:317:493)
    at W (chrome-extension://jkoeaghipilijlahjplgbfiocjhldnap/js/npm.scheduler.js:21:197)
    at MessagePort.G.port1.onmessage (chrome-extension://jkoeaghipilijlahjplgbfiocjhldnap/js/npm.scheduler.js:18:296)</pre>

## Build information:

- Version: 1.26.1
- NODE_ENV: production
- STORYBOOK: false
- target: chromium
- build: stable
- architecture: web
- firefoxVariant: false
- resolution: desktop
- BUILD_DATE: 2021-01-29T07:24:15.190Z
- VERSION: v1.26.1

## Git ():

4f052038 (released) on tag "v1.26.1"
git@github.com:DimensionDev/Maskbook.git

**Comment by septs** [comment]
https://github.com/DimensionDev/Maskbook/blob/56f53a3e276ed00dc76f42a8c9ea39f729979f35/packages/maskbook/src/plugins/ITO/define.tsx#L48-L63

**Comment by guanbinrui** [comment]
![test](https://user-images.githubusercontent.com/52657989/106655532-7415f400-65d4-11eb-9b48-4d5a559dfeec.png)

I was stuck in the loading stage :/

**Comment by Jack-Works** [comment]
@septs it won't be closed by #2340, you have to provide `<Suspense>` fallback whenever you throw a Promise.

`ITOLoading` has this line in its code

```js
const PoolBackground = getAssetAsBlobURL(new URL('../assets/pool-loading-background.jpg', import.meta.url))
```

So it has to be wrapped with `Suspense` too. (And that already in `MaskbookPluginWrapper` I just check out the code).

I will mark this issue as already fixed by this line https://github.com/DimensionDev/Maskbook/blob/master/packages/maskbook/src/plugins/MaskbookPluginWrapper.tsx#L55
